docs/server-worker-extra-args

```diff
@@ -0,0 +1,18 @@
+Add the following to you main.tf and modify the extra arguements to your liking;   
+  
+  k3s_extra_server_args = [
+    "--write-kubeconfig-mode 644",
+    "--kube-apiserver-arg default-not-ready-toleration-seconds=30",
+    "--kube-apiserver-arg default-unreachable-toleration-seconds=30",
+    "--kube-controller-arg node-monitor-period=20s",
+    "--kube-controller-arg node-monitor-grace-period=20s",
+    "--kubelet-arg node-status-update-frequency=5s",
+    #"--debug",
+    #"--oidc-issuer-url ",
+    #"--oidc-client-id",
+    #"--oidc-username-claim"
+  ]
+
+  k3s_extra_worker_args = [
+    "--kubelet-arg node-status-update-frequency=5s"
+  ]
```

Comment:
I would do the same thing I mentioned above with the code snippet and note with a link to the docs here too.